x.toml

```diff
@@ -341,7 +347,7 @@ mark-changed = []
 
 [[determinator.path-rule]]
 # Required by get_stdlib_script_abis in transaction-builder-generator.
-globs = ["language/diem-framework/compiled/transaction_scripts/abi/**/*"]
+globs = ["language/diem-framework/releases/releases/current/script_abis/**/*"]
```

Comment:
What's going on with this double path thing? Why is there a `releases/releases`?

Reply:
The outer "releases" is the crate root and the inner one is where all the artifacts are stored.

Reply:
Maybe rename the inner directory? I would have zero idea that is the case based on the names

Reply:
I used to name the inner one "versions" but found that confusing. Any suggestions? How about "artifacts"?